src/components/ListLayout.jsx

```diff
@@ -171,6 +206,8 @@ const ListLayout = ({ items, localToken }) => {
         >
           <ImCross />
         </button>
+        <br />
+        <button onClick={submitDataToDb}>Submit checked items</button>
```

Comment:
For some reason I'm not seeing this button?

Reply:
Nevermind - I see it locally, but for some reason it wasn't showing up in the preview link. Weird..

Reply:
> Nevermind - I see it locally, but for some reason it wasn't showing up in the preview link. Weird..
yeap, that is weird, I don't see it too